### PR TITLE
[ui][fix] fix logout tests to reflect UI changes

### PR DESCRIPTION
**Pull request by mcada**
//test: `@logout-test`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 1]
```diff
@@ -97,10 +100,12 @@
     @Autowired
     private CalendarUtils calendarUtils;
 
-    @And("^.*logs? out from Syndesis")
+    @And("^log out from Syndesis")
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: `And` -> `When`

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 2]
```diff
@@ -68,6 +37,34 @@
...
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Condition.matchText;
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+import static io.syndesis.qe.wait.OpenShiftWaitUtils.waitFor;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.assertj.core.data.MapEntry.entry;
```
[the remark this answers is not in the document]
Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: I imported our team settings into IDE, thought that it works correctly. I will recheck it once again.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (avano) [reply ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: @mcada any news?

[the remark this answers is not in the document]
Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: after restarting the project it now works again, but I am not sure why it stopped working

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**